pkg/discovery/contextresource.go

```diff
@@ -38,5 +38,11 @@ func GetContextResourcePath(wd string) (string, error) {
 		return composeManifestPath, nil
 	}
 
+	k8sManifestPath, err := GetK8sManifestPath(wd)
+	if err == nil {
+		oktetoLog.Infof("context will load from %s", k8sManifestPath)
+		return k8sManifestPath, nil
+	}
+	// ErrOktetoManifestNotFound belongs to the discovery module and is not the same as ErrManifestNotFound error set at okteto custom errors module
```

Comment:
Added this comment because it was not obvious for a moment. I got confused thinking this was the error at oktetoErros. Eventhough it should have the oktetoErrors module if it was, it was confusing to have both errors in my mind :/

Reply:
Not sure about this, according to the function comment, it says `returns the file that will load the context resource from`. What is the context resources and how we get it from the k8s file? 

We also have `GetHelmChartPath`, I guess it doesn't work in that case either. The purpose of having a discoverability package was to have a single place where we load everything but it seems we are loading manifests from 2 different places. I think this will give us more headaches.

I don't think we have to load the k8s manifest just for the context. I think we have to do deeper research on this. If I understood correctly your comment in the PR description, the deploy works because, although we get this same error, we are finding the file in another way